src/Surfnet/StepupRa/RaBundle/Form/Type/AmendRegistrationAuthorityInformationType.php

```diff
@@ -52,7 +52,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => 'Surfnet\StepupRa\RaBundle\Command\AmendRegistrationAuthorityInformationCommand'
+            'data_class' =>
```

Comment:
afaik we use different commands (as it was) for HTTP (controller) -> service and for service -> middleware call. Any reason to deviate here? (esp. since you added the assertions to the RaBundle command)